tools/python_api/test/test_scan_pandas.py

```diff
@@ -59,7 +58,8 @@ def test_scan_pandas(get_tmp_path):
             np.timedelta64(1000000000, 'ns'),
             np.timedelta64(2500000000, 'ns'),
             np.timedelta64(3022000000, 'ns')
-        ]
+        ],
+        'name': [None, 'Ascii only', 'ñ中国字', '😂'],
```

Comment:
check empty string as well and maybe nested string list too?

Reply:
We don't support python list right now.